applications/SystemIdentificationApplication/python_scripts/utilities/expression_utils.py

```diff
@@ -41,4 +41,19 @@ def GetContainerExpression(model_part: Kratos.ModelPart, data_location: Expressi
         KratosOA.PropertiesVariableExpressionIO.Read(expression, variable)
     else:
         raise RuntimeError(f"Unsupported {data_location}.")
-    return expression
\ No newline at end of file
+    return expression
+
+class IntervalBounder:
+    def __init__(self, bounds: list[float]) -> None:
```

Comment:
Subscriptable type hinting is a Python 3.9 feature, and it looks like we're supposed to keep supporting 3.8 for now ... 

```suggestion
    def __init__(self, bounds: "list[float]") -> None:
```